newlib/libc/machine/riscv/memcpy.S

```diff
@@ -0,0 +1,295 @@
+/* Copyright (c) 2019  SiFive Inc. All rights reserved.
```

Comment:
This copyrigth notice is clearly outdated. If you do significant changes, you should add yourself.